### PR TITLE
CY-296-Key-action-snackbar

### PR DESCRIPTION
**Pull request by nicolasshiken**

### CODE REVIEW

File: src/components/snackbar/key-action-snackbar/styles/SKeyActionSnackbarKey.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import styled, { css } from "styled-components";
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../../constants/styles/mediaquerys";
+
+interface ISKeyActionSnackbarKey {}
+
+const Bronze = css<ISKeyActionSnackbarKey>`
+  height: 16px;
+  padding: 0 1.5px;
+  background-color: #f7f9fa;
+  font-size: 11px;
```
Comment (MoritzGruber) [inline src/components/snackbar/key-action-snackbar/styles/SKeyActionSnackbarKey.tsx]: @nicolasshiken  is should be a predefined font var to use

Comment (nicolasshiken) [reply src/components/snackbar/key-action-snackbar/styles/SKeyActionSnackbarKey.tsx]: I found an issue with how the fonts are created and created a ticket [here](https://linear.app/caisy/issue/CY-396/bug-fonts-have-incorrect-formatnaming-on-design-tokens), I'll finish this ticket first so I can use font vars wherever are needed!

Comment (nicolasshiken) [reply src/components/snackbar/key-action-snackbar/styles/SKeyActionSnackbarKey.tsx]: Just updated this with the new CSSVariables

**Comment by MoritzGruber** [comment]
merged and resolved import

**Pull request merged**